xml/art_raspberry-pi.xml

```diff
@@ -288,15 +203,13 @@
    <sect3 xml:id="sec-rpi-platform-software-yast">
     <title>&yast;</title>
     <para>
-     &yast; is the installation and configuration framework for &sle;. It is
-     popular for its ease of use, flexible graphical interfaces and the capability
-     to customize your system quickly during and after installation. &yast;
-     can be used to configure your entire system: You can configure hardware,
-     set up networking, manage system services, and tune your security settings.
-     All these tasks can be accessed from the &yast; control center. To start
-     it, choose &yast; in the menu or run the command <command>xdg-su -c
-     yast2</command>. You will be prompted to enter the password of the
-     &rootuser; user.
+     &yast; is a user-friendly installation and configuration framework for
+     &sle;. &yast; can be used to configure your entire system: You can
+     configure hardware, set up networking, manage system services, and tune
+     your security settings. To start &yast;, choose &yast; in the menu or run
+     the command
+     <command>xdg-su -c yast2</command>. Enter the password of the &rootuser;
+     user when prompted.
```

Comment:
This is not entirely accurate since ~SP1, in that by default the user will not have the IceWM desktop anymore. We would at least need to prefix this with "If you installed a desktop," and/or mention an alternative for the default text-mode JeOS installation without `xdg-su`.

Reply:
What is the alternative to `xdg-su`? `sudo yast2`?